CHANGELOG.md

```diff
@@ -1,4 +1,12 @@
+## 1.5.1
```

Comment:
Yeah.  It seems like kind of a silly thing to do a Y for so I'm good with this.  Maybe we can get one more person to weigh in, then :ship: